source/_docs/z-wave/adding.markdown

```diff
@@ -38,7 +42,8 @@ Each individual value in the defined key can be anywhere from 0x00 to 0xFF. Defi
 
 ### Network Key
 
-An easy script to generate a random key:
+An easy UNIX script to generate a random key:
+(remember you can run this in Home Assistant! just add the Terminal Add-on)
```

Comment:
Changed to include the installation type names, removed the exclamation mark in the middle of the sentence and removed `just`, as is might not be `just` as easy for everybody).

```suggestion
(remember you can run this in Home Assistant OS or Supervised, you can use the Terminal Add-on)
```

Reply:
Thank you! updating in the next commit